Formula/vdirsyncer.rb

```diff
@@ -74,6 +74,45 @@ def install
     bin.env_script_all_files(libexec/"bin", :PYTHONPATH => ENV["PYTHONPATH"])
   end
 
+  def caveats
+    "The vdirsyncer launch script defaults to running every 60s with en_US.UTF-8 locale."
```

Comment:
Would be great if you could talk to upstream whether they want to include a `plist` file or what values they recommend for the parameters. Thanks!